src/styles.scss

```diff
@@ -13,28 +10,156 @@
 @import 'tailwindcss/utilities';
 
 @layer components {
-  .max-w-container {
-    @apply max-w-3xl mx-auto px-4 md:max-w-4xl md:px-2 xl:max-w-5xl xl:px-0;
+  .focus-ring {
+    @apply focus:outline-none focus:ring focus:ring-primary;
   }
+}
+
+:root {
+  --gradient: linear-gradient(
+    145deg,
+    theme('colors.primary.light'),
+    theme('colors.primary.DEFAULT'),
+    theme('colors.primary.dark')
+  );
+}
+
+body.dark {
+  --primary-light: #55f3de;
+  --primary: #55b9f3;
+  --primary-dark: #556af3;
+
+  --canvas: theme('colors.gray.950');
+  --canvas-opac: #141922dd;
+  --canvas-light: theme('colors.gray.900');
+  --canvas-shade: theme('colors.gray.800');
+
+  --text: theme('colors.gray.100');
+  --text-light: theme('colors.gray.300');
+  --text-shade: theme('colors.gray.400');
+
+  --code-function: theme('colors.orange.300');
+  --code-variable: theme('colors.green.300');
+  --code-class: theme('colors.yellow.300');
+  --code-keyword: var(--primary);
+  --code-comment: theme('colors.gray.600');
+}
+
+body:not(.dark) {
+  --primary-light: #55f3de;
+  --primary: #556af3;
+  --primary-dark: #55b9f3;
+  --canvas: theme('colors.gray.50');
+  --canvas-opac: #edf2f7dd;
+  --canvas-light: theme('colors.white');
+  --canvas-shade: theme('colors.gray.300');
+
+  --text: theme('colors.gray.800');
+  --text-light: theme('colors.gray.700');
+  --text-shade: theme('colors.gray.500');
+
+  --code-function: theme('colors.orange.600');
+  --code-variable: theme('colors.green.600');
+  --code-class: theme('colors.yellow.500');
+  --code-keyword: var(--primary);
+  --code-comment: theme('colors.gray.600');
+}
+
+* {
+  @apply transition-colors duration-100 ease-in;
+}
+
+body {
+  @apply bg-canvas text-color overflow-x-hidden;
+}
+
+.card {
+  @apply rounded-xl shadow-md bg-canvas-light overflow-y-hidden;
+  width: 280px;
+  height: 360px;
+}
+
+h3,
+h4,
+h5,
+h6,
+p,
+span {
+  @apply text-color;
+}
+
+h1,
+h2 {
+  @apply text-primary;
+}
+
+a {
+  @apply text-primary no-underline hover:text-primary-light;
+}
+
+h1  {
+  @apply text-4xl font-bold md:text-5xl;
+}
+
+h2 {
+  @apply font-semibold text-2xl;
+}
+h3 {
+  @apply font-medium text-xl;
+}
+
+h4 {
+  @apply font-medium text-lg;
+}
+
+ul {
+  @apply list-disc;
+}
+
+ol {
+  @apply list-decimal;
+}
 
-  .responsive-container {
-    @apply max-w-3xl mx-auto flex flex-col px-4 md:max-w-4xl md:px-2 xl:max-w-5xl xl:px-0;
+ul,
+ol {
+  li:before {
```

Comment:
Might be nice to move the styles inside

Outside
![Screenshot 2021-03-26 at 09 27 08](https://user-images.githubusercontent.com/8985933/112603572-7d258380-8e15-11eb-8551-e8bab1e3049a.png)

Inside
![Screenshot 2021-03-26 at 09 26 40](https://user-images.githubusercontent.com/8985933/112603602-83b3fb00-8e15-11eb-999b-91c5550fa333.png)


```suggestion
ol {
  @apply list-inside;
  li:before {
```